packages/dd-trace/src/metrics.js

```diff
@@ -49,6 +49,7 @@ module.exports = {
     }
 
     client = new Client({
+      tracingUrl: config.url || new URL(`http://${config.hostname || 'localhost'}:${config.port}`),
```

Comment:
This is unrelated to tracing other than sharing the endpoint.

Reply:
Yeah I was hoping for a better name here. Any suggestions? Maybe `agentUrl` or `dogstatsdProxyUrl`?